pkg/detectors/appointedd/appointedd.go

```diff
@@ -21,7 +22,7 @@ var (
 	client = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"appointedd"}) + `\b([a-zA-Z0-9=+]{88})`)
+	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"appointedd"}) + `\b([a-zA-Z0-9=+]{87}=)`)
```

Comment:
Might not always have base64 padding?

Reply:
I think it always is base64 padded. I just created a new one and it's b64. The test secret we have on deck is also b64. Was there another case you were think of?

Reply:
reverting back to the original